lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2382,17 +2616,18 @@ prop_balanceTransactionBalanced (Wallet' utxo wal pending) (ShowBuildable partia
     lovelaceFromCardanoTxOutValue (TxOutAdaOnly _ coin) = coin
     lovelaceFromCardanoTxOutValue (TxOutValue _ val) = selectLovelace val
 
-    withAlonzoBod
-        :: SealedTx
-        -> (Cardano.TxBody Cardano.AlonzoEra -> a)
-        -> a
-    withAlonzoBod (cardanoTx -> Cardano.InAnyCardanoEra Cardano.AlonzoEra tx) f =
-        let Cardano.Tx bod _ = tx
-        in f bod
-    withAlonzoBod _ _ = error "withBod: other eras are not handled yet"
 
     (_, nodePParams) = mockProtocolParametersForBalancing
 
+withAlonzoBod
+    :: SealedTx
+    -> (Cardano.TxBody Cardano.AlonzoEra -> a)
+    -> a
+withAlonzoBod (cardanoTx -> Cardano.InAnyCardanoEra Cardano.AlonzoEra tx) f =
+    let Cardano.Tx bod _ = tx
+    in f bod
+withAlonzoBod _ _ = error "withBod: other eras are not handled yet"
```

Comment:
Question: what is a "bod"?

Reply:
Short for "body", although now that you point it out, saving one character doesn't seems questionable 😅 . Renaming...